### ISSUE TITLE
[Bug]: [BAD_HTTP_STATUS: 403] Potential causes: IP block or streaming URL deciphering failed

### ISSUE THREAD

**Issue by Baqdawlett** [FreeTubeApp/FreeTube#6791]
### Guidelines

- [x] I have encountered this bug in the [latest release of FreeTube](https://github.com/FreeTubeApp/FreeTube/releases).
- [x] I have encountered this bug in the [official downloads of FreeTube](https://github.com/FreeTubeApp/FreeTube#official-downloads).
- [x] I have searched the issue tracker for [open](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aopen+is%3Aissue) and [closed](https://github.com/FreeTubeApp/FreeTube/issues?q=is%3Aissue+is%3Aclosed) issues that are similar to the bug report I want to file, without success.
- [x] I have searched the [documentation](https://docs.freetubeapp.io/) for information that matches the description of the bug I want to file, without success.
- [x] This issue contains only one bug.

### Describe the bug

1. Happens not that often
2. If i go back and get back to the video it helps

### Expected Behavior

3. I'm using it in Malaysia, idk probably that is the answer

### Issue Labels

accessibility issue

### FreeTube Version

latest

### Operating System Version

Microsoft Windows [Version 10.0.26100.3037]

### Installation Method

.apk (Alpine Linux Package)

### Primary API used

Local API

### Nightly Build

- [x] I have encountered this bug in the latest [nightly build](https://docs.freetubeapp.io/development/nightly-builds).

**Comment by peterweissbier** [comment]
no, its not only you getting this. i get this too in germany. maybe another fix on youtubes side to make it harder to use programs like this.

**Comment by efb4f5ff-1298-471a-8973-3d47447115dc** [comment]
Duplicate of https://github.com/FreeTubeApp/FreeTube/issues/6701
